### ISSUE TITLE
Wrong signedness for bit-fields

### ISSUE THREAD

**Issue by aarnez** [susematz/x86-64-ABI#4]
The subsubsection "Data Representation" -> "Aggregates and Unions" -> "Bit-fields" contains the following text:

    Bit-fields that are neither signed nor unsigned always have non-negative values.[ ...]

And consequently the figure "Bit-Field Ranges" shows unsigned ranges for these types of bit fields.

This is not in sync with current GCC.  In fact, GCC assigns the signedness of the underlying type to a bit-field.  For instance, an 'int' bit-field of size 1 has a value range from -1 to 0.

**Comment by susematz** [comment]
Indeed.  I've started a thread on x86-64-abi@googlegroups.com to assess wether any compilers
adhere to the psABI language (I hope none do, as they'd be incompatible with GCC).  I've checked
ICC, clang and tinycc.  If no negative reaction comes forth I'm going to change the psABI to say what
is implemented de-facto.
